### DISCUSSION TITLE
Can we permanently disable spooky warnings about about a specific entity reported as: "Unknown entities used in"?

### DISCUSSION THREAD

**Discussion by ovizii** [frenck/spook#868] [Q&A] [answered]
I am using the [samba backup](https://github.com/thomasmauerer/hassio-addons/tree/master/samba-backup) add-on which mentions in its [documentation ](https://github.com/thomasmauerer/hassio-addons/blob/master/samba-backup/DOCS.md) that: 

> This add-on includes a sensor for Home Assistant which reflects the current status and additionally has some useful statistics within its attributes. No configuration is necessary in order to use the sensor. The name of the sensor is sensor.samba_backup.

and 

> There is a known limitation that the sensor will be unavailable if you restart Home Assistant. This is caused by the way Home Assistant handles sensors which are not backed up by an entity, but instead come from an add-on or AppDaemon. You can easily fix that with the following blueprint:

The reason for my question is that with every HA reboot spooky reports Unknown entities used in and I dutifully press ignore, but every reboot brings back the same warning.

**Comment by frenck** [comment, marked as answer]
This add-on uses a method it shouldn't use. Please report the issue upstream.

The correct way to integrate with Home Assistant is by using an integration or a protocol like MQTT. This add-on bypasses everything by directly injecting stuff into Home Assistant's state machine.

It will lead to more issues.

../Frenck

**Reply by ovizii** [reply]
Thanks for having a look. I understand and [will report it](https://github.com/thomasmauerer/hassio-addons/issues/193).
